support-frontend/app/services/HttpIdentityService.scala

```diff
@@ -180,7 +180,7 @@ class IdentityService(apiUrl: String, apiClientToken: String)(implicit wsClient:
       wsClient.url(s"$apiUrl/$endpoint")
         .withHttpHeaders(headers: _*)
         .withQueryStringParameters(parameters: _*)
-        .withRequestTimeout(1.second)
+        .withRequestTimeout(3.seconds)
```

Comment:
we were getting occasional timeouts, and 1s seemed a bit tight for idapi responses.